src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -46,6 +46,7 @@
 				:id="id"
 				:token="token"
 				:can-react="canReact"
+				:is-hovered="isHovered"
```

Comment:
```suggestion
				:show-controls="isHovered || isFollowUpEmojiPickerOpen"
```

Then it won't dissapear, if you open emoji picker and move cursor out of its borders
And naming could be adjusted a bit